app/src/main/res/layout/fragment_holiday_calendar.xml

```diff
@@ -15,46 +15,103 @@
     </data>
     <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:background="@color/colorPrimary"
+        android:layout_height="match_parent"
         >
-        <ImageButton
-            android:id="@+id/button_previous_year"
-            android:layout_width="@dimen/dp_50"
-            android:layout_height="@dimen/dp_30"
-            android:layout_centerVertical="true"
-            android:background="@drawable/md_transparent"
-            android:scaleType="centerCrop"
-            android:tint="@color/color_white"
-            app:srcCompat="@drawable/ic_menu_left"
-            tools:ignore="ContentDescription"
+        <RelativeLayout
+            android:id="@+id/relative_control"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/colorPrimary"
+            >
+            <ImageButton
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_30"
+                android:layout_centerVertical="true"
+                android:background="@drawable/md_transparent"
+                android:scaleType="centerCrop"
+                android:tint="@color/color_white"
+                app:srcCompat="@drawable/ic_menu_left"
+                tools:ignore="ContentDescription"
+                />
+            <TextView
```

Comment:
WARNING: [RelativeOverlap] `TextView-3` can overlap `ImageButton-5` if TextView-3 grows due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.